Flea/Shared/NavTab.razor

```diff
@@ -1,9 +1,11 @@
+@using System.Security.Cryptography.X509Certificates
+@using System.Runtime.CompilerServices
```

Comment:
Unused imports (`@using` directives) should be removed. (Not really that important)

Reply:
Deleted